internal/common/conversion/type_conversion_test.go

```diff
@@ -67,3 +68,31 @@ func TestMongoDBRegionToAWSRegion(t *testing.T) {
 		}
 	}
 }
+
+func TestSlice_SameBehavior(t *testing.T) {
+	tests := []struct {
+		ptr   *[]string
+		slice []string
+	}{
+		{nil, nil},
+		{&[]string{"hello", "there"}, []string{"hello", "there"}},
+	}
+	for _, test := range tests {
+		assert.Equal(t, test.slice, conversion.SlicePtrToSlice(test.ptr))
+		assert.Equal(t, test.ptr, conversion.NonEmptySliceToSlicePtr(test.slice))
```

Comment:
you are testing two unrelated functions in the same test, please use `Test<FuncName>` as best practice, ie `TestSlicePtrToSlice` and `TestNonEmptySliceToSlicePtr` as separate tests

Reply:
mentioning you in the comment with the explanation

Reply:
at then end I'm only having one method, changed to follow your suggestion